ironfish/src/consensus/consensus.ts

```diff
@@ -43,9 +43,11 @@ export const MAX_SYNCED_AGE_MS = 60 * 1000
 export const MAX_REQUESTED_BLOCKS = 50
 
 /**
- * The network has a target block time of 15 seconds
- *
- * NOTE: This is not used in target calculation, or IRON_FISH_YEAR_IN_BLOCKS.
- * Ideally, it should be in a future refactor.
+ * The network has a target block time of 60 seconds
```

Comment:
It showed that IRON_FISH_YEAR_IN_BLOCKS was not used anywhere

Reply:
oh. that was a good note actually. 

because we're changing block time, IRON_FISH_YEAR_IN_BLOCKS also has to change. 
if you look in consensus.ts: 

```
/*
 * A ratio of blocks per year that represents an approximation of how many blocks are considered a "year".
 * It's generally an approximation based on 15 second block times. It's used in calculating how much a miner
 * should get in rewards. Approximately (365 * 24  * 60 * 60) / TARGET_BLOCK_TIME_MS
 */
export const IRON_FISH_YEAR_IN_BLOCKS = 2100000
```

Reply:
IRON_FISH_YEAR_IN_BLOCKS changes the miner's reward, so very important that this change also updates that. 

that was an excellent note that you deleted fwiw — i would have totally forgotten about that.